src/utils/gaa.js

```diff
@@ -481,7 +480,7 @@ export class GaaMeteringRegwall {
    * Returns the GAA user, after the user signs in.
    * @private
    * @nocollapse
-   * @return {!Promise<!GoogleUserDef>}
+   * @return {!Promise<!GaaUserDef>}
```

Comment:
Closure compiler might be better at catching type issues when using async/await? Not sure why it missed this before